service/s3/proto_api_op_GetObject_test.go

```diff
@@ -0,0 +1,52 @@
+package s3_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/internal/awstesting/mock"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestProtoGetObjectRequest_Send(t *testing.T) {
+	svc := s3.New(mock.Config())
+	input := s3.GetObjectInput{
+		Bucket:                     aws.String("mock bucket"),
+		IfMatch:                    nil,
+		IfModifiedSince:            nil,
+		IfNoneMatch:                nil,
+		IfUnmodifiedSince:          nil,
+		Key:                        aws.String("mock key"),
+		PartNumber:                 nil,
```

Comment:
Would be helpful to fill out these values for this test, and future iterations of this protototype.